src/widgets.js

```diff
@@ -343,9 +468,15 @@ define([
                 }
             }, !!widget.isDisabled());
         };
+        
+        if (mug.options.richtext) {
+            insertTpl = '<span class="label label-datanode label-datanode-internal" contenteditable=false draggable=true data-value="${name}"><i class="${icon}">&nbsp;</i>${name}<i class="close">&times;</i></span>';
+        }
 
-        util.questionAutocomplete(widget.input, mug,
-                                  {property: options.path});
+        util.questionAutocomplete(widget.input, mug, {
+            property: options.path,
+            insertTpl: insertTpl,
```

Comment:
Little weird, but this does pass `undefined` when `richtext == false`. That allows `_.defaults` to overwrite it in src/util